src/index.ts

```diff
@@ -14,7 +14,10 @@ program.option(
 );
 program.option("--account <email>", "the Google account to use for authorization");
 program.option("-j, --json", "output JSON instead of text, also triggers non-interactive mode");
-program.option("--token <token>", "supply an auth token for this command");
+program.option(
+  "--token <token>",
+  "DEPRECATED - will be removed in a future major version - supply an auth token for this command"
+);
```

Comment:
Technically `--token` accepts both refresh tokens and access tokens. I'd like us to stop accepting refresh tokens (because they are long-lived user-attached bearer tokens), but I'm not 100% sure I want to stop accepting access tokens (since they are short-lived).

Maybe we should instrument how often `--token` gets used with access tokens to help decide.

Reply:
I'm not sure I could tell the difference between the two, but... maybe. IMO, I think SAs are cleaner - and if we're not giving a way to generate or show that token (it would have to be yoinking the CLI token or using gcloud?), I'm disinclined to keep the flag around for (what I would guess is) a small number of use cases.